tests/msc3083_test.go

```diff
@@ -159,3 +162,159 @@ func TestRestrictedRoomsRemoteJoin(t *testing.T) {
 	// Execute the checks.
 	CheckRestrictedRoom(t, alice, bob, space, room)
 }
+
+// Request the room summary and ensure the expected rooms are in the response.
+func RequestAndAssertSummary(t *testing.T, user *client.CSAPI, space string, expected_rooms []interface{}) {
```

Comment:
Keep these private please, there is no need to make it public.

Reply:
Thanks! I also made the other helpers in this file private.